nilearn/glm/tests/test_thresholding.py

```diff
@@ -14,7 +14,7 @@
 def test_fdr():
     rng = np.random.RandomState(42)
     n = 100
-    x = np.linspace(.5 / n, 1. - .5 / n, n)
+    x = np.linspace(1 / n, 1, n)
```

Comment:
Thx, but the idea is to have a test such that it fails with the previous version, and passes after the fix.